### PR TITLE
added doc for mapper

### PR DESCRIPTION
**Pull request by AdityaSoni19031997**
Need to add the return type of mapper also..
But was not sure whether it's a dict or a list or something else..

### CODE REVIEW

**Comment by jph00** [comment]
Your docs aren't quite right. Try running the code and make sure you understand how it works. You'll be able to see the exact return type too.

Basically the idea is: when training, you don't pass in a mapper, but one gets calculated, containing the means and standard deviations of the continuous variables in the training set. Then, for the test set, you pass in that mapper, so that you scale the test set in the same way.

**Comment by AdityaSoni19031997** [comment]
updated

**Comment by AdityaSoni19031997** [comment]
https://stackoverflow.com/a/40352356

**Comment by AdityaSoni19031997** [comment]
The DataFrameMapper transformation tuple describes which columns should be run through the transformation.

**Comment by AdityaSoni19031997** [comment]
edited again..
